src/core/mdn-annotation.js

```diff
@@ -86,8 +86,8 @@ function buildBrowserSupportTable(support) {
   let innerHTML = "";
   function addMDNBrowserRow(browserId, yesNoUnknown, version) {
     const displayStatus = yesNoUnknown === "Unknown" ? "?" : yesNoUnknown;
-    const classList = `${browserId} ${yesNoUnknown.toLowerCase()}`;
-    const browserRow = `
+    const classList = hyperHTML `${browserId} ${yesNoUnknown.toLowerCase()}`;
```

Comment:
This specific line is okay to be string, because it will be escaped by the next line.